### PR TITLE
feat(trace): allow navigating from error to source

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: packages/playwright/src/common/ipc.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ export type ConfigCLIOverrides = {
   timeout?: number;
   ignoreSnapshots?: boolean;
   updateSnapshots?: 'all'|'none'|'missing';
-  workers?: number;
+  workers?: number | string;
```
Comment (dgozman) [inline packages/playwright/src/common/ipc.ts:36]: This looks unrelated!

**Review by dgozman — APPROVED** [review]

File: packages/playwright/src/common/ipc.ts (hunk 1 above)
Comment (pavelfeldman) [reply packages/playwright/src/common/ipc.ts:36]: but it is correct!
